frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -139,9 +134,38 @@ async function verifyIdentity(userId: string, domain: string) {
 }
 
 async function verifyForestClientNumber(forestClientNumber: string) {
-    forestClientData.value = (
-        await forestClientApi.search(forestClientNumber)
-    ).data;
+    const forestNumbers = forestClientNumber.split(/[ ,]+/);
+
+    for (const item of forestNumbers) {
+        if (isNaN(parseInt(item))) return;
+        await forestClientApi.search(item).then((result) => {
+            if (
+                result.data[0].status?.status_code ===
```

Comment:
I think this is where the error comes from for invalid forest client. If not found, we need to handle it differently